org.hl7.fhir.utilities/src/main/resources/source/rendering-phrases-es.po

```diff
@@ -1,5480 +1,5486 @@
+msgstr ""
+
 # ADD_BIND_UI
 #: ADD_BIND_UI
-msgctxt "ADD_BIND_UI"
-msgid "UI"
-msgstr ""
-
+msgctxt "ADD_BIND_UI"
+msgid "UI"
+msgstr ""
+
 # ADD_BIND_UI_BIND
 #: ADD_BIND_UI_BIND
-msgid "UI Binding"
-msgstr ""
-
+msgid "UI Binding"
+msgstr ""
+
 # ADD_BIND_UNKNOWN_PUR
 #: ADD_BIND_UNKNOWN_PUR
-msgid "Unknown code for purpose"
-msgstr ""
-
+msgid "Unknown code for purpose"
+msgstr ""
+
 # ADD_BIND_VALID_EXT
 #: ADD_BIND_VALID_EXT
-msgid "Validators will check this binding (strength = extensible)"
-msgstr ""
-
+msgid "Validators will check this binding (strength = extensible)"
+msgstr ""
+
 # ADD_BIND_VALID_REQ
 #: ADD_BIND_VALID_REQ
-msgid "Validators will check this binding (strength = required)"
-msgstr ""
-
+msgid "Validators will check this binding (strength = required)"
+msgstr ""
+
 # ADD_BIND_VALUE_COMP
 #: ADD_BIND_VALUE_COMP
-msgid "This value set is a component of the base value set"
-msgstr ""
-
+msgid "This value set is a component of the base value set"
+msgstr ""
+
 # BUNDLE_ETAG
 #: BUNDLE_ETAG
-msgid "ETag = {0}"
-msgstr "ETag = {0}"
-
+msgid "ETag = {0}"
+msgstr "ETag = {0}"
+
 # BUNDLE_HEADER_DOCUMENT_CONTENTS
 #: BUNDLE_HEADER_DOCUMENT_CONTENTS
-msgid "Additional Resources Included in Document"
-msgstr ""
-
+msgid "Additional Resources Included in Document"
+msgstr ""
+
 # BUNDLE_HEADER_ENTRY
 #: BUNDLE_HEADER_ENTRY
-msgid "Entry {0}"
-msgstr ""
-
+msgid "Entry {0}"
+msgstr ""
+
 # BUNDLE_HEADER_ENTRY_URL
 #: BUNDLE_HEADER_ENTRY_URL
-msgid "Entry {0} - fullUrl = {1}"
-msgstr ""
-
+msgid "Entry {0} - fullUrl = {1}"
+msgstr ""
+
 # BUNDLE_HEADER_ROOT
 #: BUNDLE_HEADER_ROOT
-msgid "Bundle {0} of type {1}"
-msgstr ""
-
+msgid "Bundle {0} of type {1}"
+msgstr ""
+
 # BUNDLE_IF_MATCH
 #: BUNDLE_IF_MATCH
-msgid "If-Match = {0}"
-msgstr ""
-
+msgid "If-Match = {0}"
+msgstr ""
+
 # BUNDLE_IF_MOD
 #: BUNDLE_IF_MOD
-msgid "If-Modified-Since = {0}"
-msgstr ""
-
+msgid "If-Modified-Since = {0}"
+msgstr ""
+
 # BUNDLE_IF_NONE
 #: BUNDLE_IF_NONE
-msgid "If-None-Exist = {0}"
-msgstr "IExist = {0}"
-
+msgid "If-None-Exist = {0}"
+msgstr ""
+
 # BUNDLE_IF_NON_MATCH
 #: BUNDLE_IF_NON_MATCH
-msgid "If-None-Match = {0}"
-msgstr ""
-
+msgid "If-None-Match = {0}"
+msgstr ""
+
 # BUNDLE_LAST_MOD
 #: BUNDLE_LAST_MOD
-msgid "LastModified = {0}"
-msgstr ""
-
+msgid "LastModified = {0}"
+msgstr ""
+
 # BUNDLE_LOCATION
 #: BUNDLE_LOCATION
-msgid "Location = {0}"
-msgstr "Lo= {0}"
-
+msgid "Location = {0}"
+msgstr ""
+
 # BUNDLE_REQUEST
 #: BUNDLE_REQUEST
-msgid "Request:"
-msgstr ""
-
+msgid "Request:"
+msgstr ""
+
 # BUNDLE_RESOURCE
 #: BUNDLE_RESOURCE
-msgid "Resource {0}:"
-msgstr "Rec}:"
-
+msgid "Resource {0}:"
+msgstr ""
+
 # BUNDLE_RESPONSE
 #: BUNDLE_RESPONSE
-msgid "Response:"
-msgstr ""
-
+msgid "Response:"
+msgstr ""
+
 # BUNDLE_REV_EXCP
 #: BUNDLE_REV_EXCP
-msgid "Exception generating narrative: {0}"
-msgstr ""
-
+msgid "Exception generating narrative: {0}"
+msgstr ""
+
 # BUNDLE_SEARCH
 #: BUNDLE_SEARCH
-msgid "Search:"
-msgstr "Búsqueda:"

```

Comment:
@vadi2 What is happening here? This looks like it's a actually deleting older, valid translations.